keyboards/sevencrumbs/matrix/info.json

```diff
@@ -0,0 +1,47 @@
+{
+    "manufacturer": "Seven Crumbs",
+    "keyboard_name": "Matrix",
+    "maintainer": "ZoidTechnology",
+    "board": "GENERIC_RP_RP2040",
+    "bootloader": "rp2040",
+    "bootloader_instructions": "Press and hold all four buttons while connecting the USB cable.",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["GP0", "GP1"],
+            ["GP2", "GP3"]
+        ]
+    },
+    "processor": "RP2040",
+    "rgb_matrix": {
+        "driver": "custom",
+        "layout": [
+            {"matrix": [0, 0], "x": 104, "y": 24, "flags": 4},
+            {"matrix": [0, 1], "x": 120, "y": 24, "flags": 4},
+            {"matrix": [1, 0], "x": 104, "y": 40, "flags": 4},
+            {"matrix": [1, 1], "x": 120, "y": 40, "flags": 4}
+        ]
+    },
+    "url": "https://zoid.com.au/matrix/",
+    "usb": {
+        "device_version": "1.0.0",
+        "max_power": 200,
+        "pid": "0x1069",
+        "vid": "0x2E8A"
```

Comment:
https://www.the-sz.com/products/usbid/index.php?v=0x2E8A&p=&n=

This is still the Raspberry Pi foundation vendor ID.  This should be changed.

Reply:
I was allocated the product ID `0x1069` under Raspberry PI's vendor ID. The details are here: https://github.com/raspberrypi/usb-pid My PID hasn't been added to the table yet, but I have received email confirmation.